thunderbird/utils.py

```diff
@@ -11,6 +11,8 @@
 import logging
 import os
 
+MAX_OCCURS = 1000
```

Comment:
I wonder if it would be better to put constants into their own importable file. Any thoughts @jameshiebert?